spec/transaction_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'transaction'
+
+describe Transaction do
+
+  before do
```

Comment:
It feels like not all the tests use both of these transactions.  Is it possible to refactor so only the necessary setup is done for each test?